regen-network/regen-web
# feat: APP-375 update /profile routes to be under /dashboard and /dashboard/admin

## Description

https://regennetwork.atlassian.net/browse/APP-375

### Author Checklist


### How to test

1. https://deploy-preview-2505--regen-marketplace.netlify.app/ 
2. Log in and verify that all links work as expected in the dashboard, edit profile side nav and main nav as stated in the Jira issue

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
